### DISCUSSION TITLE
Can't seem to find how to mimic loading state

### DISCUSSION THREAD

**Discussion by danielvanmil** [zendesk/laika#38] [General]
Hello,

Doc says "simulate loading state", but I can't seem to find out how?

Can anyone help me?

**Comment by niieani** [comment]
Hi @danielvanmil. If you want to simulate loading state, setup an interceptor, [disable network fallback](https://zendesk.github.io/laika/docs/api/classes/Laika.InterceptApi#disablenetworkfallback), and don't mock the result until after you want it to be sent. That should work, though we don't have a test for it. Accepting PRs for a doc update and/or the test.
